airbyte_cdk/cli/airbyte_cdk/_manifest.py

```diff
@@ -19,6 +31,176 @@ def manifest_cli_group() -> None:
     pass
 
 
+@manifest_cli_group.command("validate")
+@click.option(
+    "--manifest-path",
+    type=click.Path(exists=True, path_type=Path),
+    default="manifest.yaml",
+    help="Path to the manifest file to validate (default: manifest.yaml)",
+)
+def validate_manifest(manifest_path: Path) -> None:
+    """Validate a manifest file against the declarative component schema.
+
+    This command validates the manifest file and checks version compatibility.
+    If validation fails, it will suggest running the migrate command if needed.
+
+    Exit codes:
+    \\b
+    0: Manifest is valid and up to date
+    1: Manifest is valid but needs migration, or general errors
+    2: Manifest has validation errors that are fixable via migration
+    3: Manifest has validation errors that are NOT fixable via migration
```

Comment:
Consolidate exit codes 1 and 2. Or define the use case more clearly for '1' as not a subset of '2'.